examples/jenkins-master/jenkins-master-template.json

```diff
@@ -48,6 +42,12 @@
       "displayName": "Jenkins Reference",
       "description": "The git ref or tag to use for customization.",
       "value": "master"
+    },
+    {
+      "name": "NAMESPACE",
+      "displayName": "Namespace",
+      "description": "The OpenShift Namespace where the ImageStream resides.",
```

Comment:
i copied the language from all our other templates, so i'm inclined to leave it at this point.